Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -77,14 +77,28 @@ namespace Unity.MeshSync
 /// </summary>
 internal delegate void DeleteInstanceHandler(string path);
 
+    public enum UpdateKind {
+        Asset,
+        Entity,
+        Instance,
+        InstanceInfo,
+        Property
+    }
+
+    public struct MeshSyncAnalyticsData {
+        internal AssetType asset_Type;
+        internal EntityType entity_Type;
```

Comment:
I don't understand, where is this enfornced in meshsync?

Everything else is using camelCase?

![image](https://user-images.githubusercontent.com/68228653/192552069-121749b1-c25d-420b-b450-2757b342ba3b.png)